theano/tensor/nnet/conv3d2d.py

```diff
@@ -328,7 +329,9 @@ def local_to_gpu(node):
                 new_inp = list(node.inputs)
                 for idx in to_gpu:
                     new_inp[idx] = cuda.gpu_from_host(new_inp[idx])
-                return [cuda.host_from_gpu(op()(*new_inp))]
+                new_node = cuda.host_from_gpu(op()(*new_inp))
+                copy_stack_trace(node.outputs[0], new_node)
```

Comment:
That's right, the variable returned by `op()(*new_inp)` should get the stack trace as well.